test/test.md

```diff
@@ -0,0 +1,15 @@
+# The Mathlib Test Directory
```

Comment:
I'd rename this file to `test/README.md` so that it gets displaeyd automatically in GitHub.

Reply:
good idea!

Reply:
good idea!